doc/cephfs/fuse.rst

```diff
@@ -1,54 +1,106 @@
-=======================
-Mount CephFS using FUSE
-=======================
+========================
+ Mount CephFS using FUSE
+========================
 
-Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the client
-host has a copy of the Ceph configuration file and a keyring with CAPS for the
-Ceph metadata server.
+Prerequisite
+------------
+Before mounting CephFS, ensure that the client host (where CephFS has to be
+mounted and used) has a copy of the Ceph configuration file (i.e.
+``ceph.conf``) and a keyring of the CephX user that has CAPS for the Ceph MDS.
+Both of these files must be present on the host where the Ceph MON resides.
 
-#. From your client host, copy the Ceph configuration file from the monitor host 
-   to the ``/etc/ceph`` directory. :: 
+#. Generate a minimal conf for the client host. The conf file should be
+   placed at ``/etc/ceph``::
 
-	sudo mkdir -p /etc/ceph
-	sudo scp {user}@{server-machine}:/etc/ceph/ceph.conf /etc/ceph/ceph.conf
+    # on client host
+    mkdir /etc/ceph
+    ssh {user}@{mon-host} "sudo ceph config generate-minimal-conf" | sudo tee /etc/ceph/ceph.conf
 
-#. From your client host, copy the Ceph keyring from the monitor host to 
-   to the ``/etc/ceph`` directory. :: 
+   Alternatively, you may copy the conf file. But the above method creates a
+   conf with minimum details which is better.
```

Comment:
"But the method which generates the minimal config is usually sufficient. For more information, see https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#bootstrap-options"